main.py

```diff
@@ -156,7 +156,8 @@ def reply(msg=None, img=None):
                         pairs.append(pair)
                 r = 'Reply with /<assetpair> to get bid/ask prices\n{}'.format(', '.join(pairs))
                 reply(r)
-            elif text[1:].upper() in ASSETPAIRS.keys():
+
+            elif text.split(' ')[0][1:] in ASSETPAIRS.keys():
                 pair = text[1:].upper()
```

Comment:
pair also needs to be adjusted to read from the first part of the split `pair = text.split(' ')[0][1:]`